src/content/developers/docs/scaling/optimistic-rollups/index.md

```diff
@@ -1,46 +1,254 @@
+description: An introduction to optimistic rollups, a scaling solution used by the Ethereum community. 
 lang: en
 sidebar: true
 ---
 
+Optimistic rollups are off-chain protocols designed to extend throughput of Ethereum's base layer. They reduce computation on the main Ethereum chain by processing transactions off-chain, offering significant improvements in processing speeds. Unlike other scaling solutions, such as [sidechains](/developers/docs/scaling/sidechains/), optimistic rollups derive security from Mainnet by publishing transaction results on-chain. 
+
+As computation is the slow, expensive part of using Ethereum, optimistic rollups can offer up to 10-100x improvements in scalability. Optimistic rollups also write transactions to Ethereum as `calldata`, reducing gas costs for users. 
+
 ## Prerequisites {#prerequisites}
 
 You should have a good understanding of all the foundational topics and a high-level understanding of [Ethereum scaling](/developers/docs/scaling/). Implementing scaling solutions such as rollups is an advanced topic as the technology is less battle-tested and continues to be researched and developed.
 
 Looking for a more beginner-friendly resource? See our [introduction to layer 2](/layer-2/).
 
-## Optimistic rollups {#optimistic-rollups}
+## What is an optimistic rollup? {#what-is-an-optimistic-rollup} 
+
+An optimistic rollup is an approach to scaling Ethereum that involves moving computation and state storage off-chain. Optimistic rollups execute transactions outside of Ethereum, but post transaction data to Mainnet as `calldata`.
+
+Optimistic rollup operators bundle multiple off-chain transactions together in large batches before submitting to Ethereum. This approach enables spreading fixed costs across multiple transactions in each batch, reducing fees for end-users. Optimistic rollups also use compression techniques to reduce the amount of data posted on Ethereum. 
+
+Optimistic rollups are considered “optimistic” because they assume off-chain transactions are valid and don't publish proofs of validity for transaction batches posted on-chain. This separates optimistic rollups from [zero-knowledge rollups](/developers/docs/scaling/zk-rollups) that publish cryptographic [proofs of validity](/glossary/#validity-proof) for off-chain transactions. 
+
+Optimistic rollups instead rely on a fraud-proving scheme to detect malicious transactions. After a rollup batch is submitted on Ethereum, there's a time window (called a challenge period) during which anyone can challenge the validity of a rollup transaction by computing a [fraud proof](/glossary/#fraud-proof). 
+
+If the fraud proof succeeds, the rollup protocol reverses the invalid block and punishes the malicious operator. If the rollup batch remains unchallenged after the challenge period elapses, it is deemed valid and accepted on Ethereum. The rollup will advance to a new state and the operator can start adding new blocks. 
```

Comment:
"If the fraud proof succeeds, the rollup protocol reverses the invalid block and punishes the malicious operator."

The invalid block doesn't get reversed. Instead, the transactions are executed correctly and the state is updated to the correct value.

Also, the rollup doesn't wait the challenge period (typically seven days) until it adds new blocks. It adds them immediately, and if there is a successful challenge then the results of all those new blocks have to be recalculated.

Reply:
So, I got this from the Paradigm research piece I quoted earlier: 

> We now can introduce the [State Commitment Chain](https://github.com/ethereum-optimism/contracts-v2/blob/0ad4dcfdef11ef87e278a8159de8414c8e329ba1/contracts/optimistic-ethereum/OVM/chain/OVM_StateCommitmentChain.sol) (SCC). The SCC contains a list of state roots, which, in the optimistic case, correspond to the result of applying each transaction in the CTC against the previous state. If this is not the case, then the fraud verification process allows the invalid state root, and all following it, to be deleted, so that the correct state root for those transactions may be proposed.

That said, I want to ask: if the invalid block is not rejected or reversed, what happens to its position in the chain? 

As to the concluding part, I changed it to read: 

> Others can continue to build on an unconfirmed rollup block, but with a caveat: blocks descending from a parent block will be rejected if the parent is later declared invalid.

Reply:
I'm tired, so I'll do the simple comments now and deal with the complex ones later this weekend, OK?

Reply:
> if the invalid block is not rejected or reversed, what happens to its position in the chain?

I'm not sure about Arbitrum, but in Optimism the block just contains transactions, and it cannot be invalid. What can be invalid is the post-block state, which is written elsewhere (in the state commitment chain, SCC). The SCC is reverted, but then it needs to be recalculated with the correct result for that block and then subsequent blocks. The transactions themselves are still part of the chain.

Reply:
I verified this with the developers. Yes, transactions are never invalid, you could say they get removed and immediately re-executed, but I think it's clearer to say their execution result is corrected.

Reply:
So, I changed the text to read: 

> If the fraud proof succeeds, the rollup protocol re-executes the transaction(s) and updates the rollup's state accordingly. The other effect of a successful fraud proof is that the sequencer responsible for including the incorrectly executed transaction in a block receives a penalty. 

> If the rollup batch remains unchallenged (i.e., all transactions are correctly executed) after the challenge period elapses, it is deemed valid and accepted on Ethereum. Others can continue to build on an unconfirmed rollup block, but with a caveat: transaction results will be reversed if based on an incorrectly executed transaction published previously. 

LMK what you think.

Reply:
Approve.

Reply:
Good. Thanks for reviewing.